dom0/sd-base-template-files.sls

```diff
@@ -21,6 +21,7 @@ sd-base-template-install-additional-packages:
 sd-base-template-install-securedrop-packages:
   pkg.installed:
     - pkgs:
+      - securedrop-qubesdb
```

Comment:
This and the `securedrop-whonix-config` package will fail in CI until freedomofpress/securedrop-client#2032 is merged.